### ISSUE TITLE
example problem raspberry

### ISSUE THREAD

**Issue by YvesBB** [timredfern/ofxHelios#1]
hello
the example "example-transforms" works well without dac,
when i plug the dac i have this message :
VER ID IS 10
using newer build and GLFW window
[notice ] ofxHelios v 0.1: found laser DAC: firmware v 5
[notice ] ofxHelios v 0.1: set intensity to 37
Segmentation fault
make: *** [/home/pi/openFrameworks/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk:184: run] Error 139

any idea??
thanks
the problem come from this line :

 colourPolyline square=makePolygon(6,400).getTransformed(m);

**Comment by timredfern** [comment]
yeah I think I know why that is.. will update when I get the chance
